usage/plugins/org.jboss.tools.usage/META-INF/MANIFEST.MF

```diff
@@ -20,7 +20,9 @@ Export-Package: org.jboss.tools.usage.branding,
  org.jboss.tools.usage.util.reader
 Require-Bundle: org.eclipse.osgi;bundle-version="[3.5.0,4.0.0)",
  org.eclipse.core.runtime;bundle-version="[3.5.0,4.0.0)",
- org.eclipse.ui;bundle-version="[3.5.0,4.0.0)"
+ org.eclipse.ui;bundle-version="[3.5.0,4.0.0)",
+ org.eclipse.epp.logging.aeri.core;bundle-version="1.100.0",
+ org.eclipse.epp.logging.aeri.ide;bundle-version="1.100.0";resolution:=optional

```

Comment:
IMO if we let it stay just optional none greedy it is not going to be installed.

Dependency to usage is declared as optional and greedy 
``` java
org.jboss.tools.usage;resolution:=optional;x-installation:=greedy
```
the reason behind greedy is to force installation if usage bundle is available from any p2 repository, because none greedy optional bundles are not installed by default and user have to install them manually as part of other feature install.